both/i18n/en-us.i18n.yml

```diff
@@ -30,7 +30,8 @@ navbar:
     transactions: 'Transactions'
     proposals: 'Proposals'
     votingPower: 'Voting Power'
-    lang: "ENG"
+    lang: "ES"
```

Comment:
This should be kept as "ENG" as this is the lang display on the navbar for English. You have already done this translation in the Spanish translation file.

Reply:
Changed, muchas gracias!

Reply:
¡Excelente!